packages/bindings/src/lib.rs

```diff
@@ -24,4 +24,7 @@ pub mod relationships;
 pub mod reports;
 #[cfg(feature = "subspaces")]
 pub mod subspaces;
+#[cfg(feature = "integration-test")]
```

Comment:
Instead of using a custom feature is better to use the `#[cfg(not(target_arch = "wasm32"))]` so that the `testing_utils` module will be automaticaly compiled when performing the integration tests.  
```suggestion
#[cfg(not(target_arch = "wasm32"))]
```

Reply:
You are right. I didn't think about it. Since it is a lib that mock desmos app for integration test, I use `mock` as its feature.